src/Arcus.Observability.Tests.Unit/Telemetry/ILoggerExtensionsTests.cs

```diff
@@ -914,6 +1050,9 @@ public void LogBlobStorageDependencyWithDependencyMeasurement_ValidArguments_Suc
             Assert.Contains(isSuccessful.ToString(), logMessage);
             Assert.Contains(startTime.ToString(FormatSpecifiers.InvariantTimestampFormat), logMessage);
             Assert.Contains(duration.ToString(), logMessage);
+            // Check specifically on dependencyName
+            string dependencyName = $"{accountName}/{containerName}";
+            Assert.Contains("Azure blob " + dependencyName, logMessage);
```

Comment:
If we move the dependency name to a dedicated variable, there's no need to specify it via a comment, right?
```suggestion
            string dependencyName = $"{accountName}/{containerName}";
            Assert.Contains("Azure blob " + dependencyName, logMessage);
```